notebooks/week1/01.dataExploraton.py

```
@@ -0,0 +1,145 @@
+
+# COMMAND ----------
+
+# Update the list of numerical features to include time series columns
+time_series_cols = train_ts.columns.tolist()
+time_series_cols.remove("id")  # Temporarily remove 'id' column from the list of time series columns
+num_features += time_series_cols
+
+# COMMAND ----------
+
+def update(df):
+    for c in cat_features: 
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)